### ISSUE TITLE
Where is the processing server launched?

### ISSUE THREAD

**Issue by aminghaderi** [sandrohanea/whisper.net#169]
Hello.
I'm sorry, I didn't understand where you run the whisper.cpp server.
And where do you download and run the language processing files, which are several gigabytes in size?
Is the processing workload running on a computer? Or is there a specific web server (like Microsoft or Google speech processing servers) that the program interacts with exclusively?
I need an independent framework that is not dependent on any other server service.
In other words, I need to connect to my personal server where I have executed whisper.cpp.
Thank you very much.

**Comment by sandrohanea** [comment]
There is no need to run whisper.cpp server, this library will use interop and will call directly whisper.cpp native methods. 

In other words: it runs fully locally where your C# code is running (in the same process)
